src/sage/misc/cachefunc.pyx

```diff
@@ -1940,7 +1955,7 @@ cdef class CachedMethodCaller(CachedFunction):
             True
         """
         if self._instance is None:
-            # cached method bound to a class
+            # cached method bound to a class i.e. unbound method, such as ``Foo.f``
```

Comment:
unbound cached method such as ``Foo.f''

(an unbound method isn't bound to a class. It's simply unbound. There are class methods, where the first argument is the class rather than the instance (important in inheritance), but this is not that)

Reply:
Not my terminology. I thought I'll keep the old comment just in case, but if people are good with replacing it entirely that's good too.

Reply:
it seems that the author might have thought that `bound to` means `connected with a dot to`. That's not what it means for methods in python.